### PR TITLE
fixes #74

### LINKED ISSUE #74: restiming causes redirect to beacon url
Hi,

I can't use the restiming plugin because there is a systematically redirect to the beacon handler when this plugin is added to the boomerang bundle.

Minimal repro (
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Bootstrap 101 Template</title>
    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
    <script src="boomerang.js"></script>
    <script src="restiming.js"></script>
    <script>
    BOOMR.init({
        beacon_url: "/boomerang_handler"
    });
</script>
  </head>
  <body>
    <h1>Hello, world!</h1>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  </body>
</html>
```

resttiming

From what I can see, there is because of the form.submit() on the child iframe created when the plugin is added to the page.

### PR DESCRIPTION
**Pull request by Cybermaxs**
Regarding #74, we can't add a form directly to a dynamic iframe.
contentWindow and contentDocument should be used.

### CODE REVIEW

**Comment by bluesmoon** [comment]
can you make sure this passes the eslint checks?  Also, make sure you're using tabs for indentation.  Thanks.  @nicjansma please review

**Comment by Cybermaxs** [comment]
Ok, sorry for that.

There is still one eslint error (unused variable in a test file) but not depending on this PR

File: boomerang.js [hunk 1]
```diff
@@ -734,8 +734,18 @@ BOOMR_check_doc_domain();
 
 					document.body.appendChild(iframe);
 
-					// Add the form to the iframe
-					iframe.appendChild(form);
+                    // Add the form to the iframe
```
Comment (bluesmoon) [inline boomerang.js]: seems like indentation on this comment line is still off.

**Comment by bluesmoon** [comment]
@andreas-marschke please verify

**Pull request merged**